src/providers/email.js

```diff
@@ -0,0 +1,13 @@
+import Provider from './provider.js';
+
+export default class extends Provider {
+  constructor(params) {
+    super(params);
+    this.name = 'Email';
+    this.icon = 'email';
+  }
+
+  get url() {
+    return `mailto:?body=https://${this.baseHost}/details/${this.identifier}&subject=${this.description} : ${this.creator}${this.promoCopy}`;
```

Comment:
QUESTION: Any escaping needed for `this.description`, for URL encoding or XSS prevention?

Reply:
I had assumed the escaping was happening on the Petabox side.

Reply:
Following the principle that data is passed around in raw-but-also-not-necessarily-trusted form internally and that it is any final renderer's job to make sure escaping happens that is appropriate to the render context, I would say you should take a moment to examine where malicious code might be entering the code under consideration and assume the worst--then handle it correctly.

Reply:
If we can perform the sanitization on the server side, why would we instead put extra code into the payload that the client needs to receive? I think a view should, whenever possible, simply take the data it's been given and render it. Any processing of said data should happen before it gets to the view layer. If this were a PHP class, we wouldn't want to mix the sanitizing code directly inline with the view rendering.

Reply:
I guess it depends on where we feel the barrier should be located, i.e. whose job sanitization should be. I don't want to hold this up needlessly, esp. given your timeline. I'm okay with this going out as-is, though I'd like to encourage not assuming anything about the health/security of the Petabox code -- it's worth taking even a moment to look back and see if there are any obvious vulnerabilities coming downstream to the view, e.g. as a general example, direct user form submission that gets rendered in a view.

Reply:
this should be handled at the Book Model level that we are setting up.
Let's move this over.

Reply:
>If we can perform the sanitization on the server side, 
we won't be as the Book Model is a JS file